UIKit/Tasks/MenuViewController.swift

```diff
@@ -5,9 +5,189 @@ import UIKit
+        button.frame = CGRect(x: 45, y: 530, width: 220, height: 19)
+        return button
+    }()
+
+    private let thirdVarMenuImage: UIImageView = {
+        let image = UIImageView()
+        image.image = UIImage(named: "rect")
+        image.frame = CGRect(x: 21, y: 680, width: 335, height: 80)
+        return image
+    }()
+
+    private let secondVarMenuImage: UIImageView = {
+        let image = UIImageView()
+        image.image = UIImage(named: "rect")
+        image.frame = CGRect(x: 20, y: 590, width: 335, height: 80)
+        return image
+    }()
+
+    private let variantsMenuImage: UIImageView = {
+        let image = UIImageView()
+        image.image = UIImage(named: "variants")
+        image.frame = CGRect(x: 20, y: 500, width: 335, height: 80)
+        return image
+    }()
+
+    private let menuImage: UIImageView = {
+        let image = UIImageView()
+        image.image = UIImage(named: "menu")
+        image.frame = CGRect(x: 120, y: 410, width: 125, height: 80)
+        return image
+    }()
+
+    private let pointViewLabel: UIImageView = {
+        let image = UIImageView()
+        image.image = UIImage(named: "po")
+        image.frame = CGRect(x: 320, y: 350, width: 20, height: 29)
+        return image
+    }()
+
+    private let textLabelPointView: UILabel = {
+        let label = UILabel()
+        label.text = "Разрѣшите доступъ къ ​геолокаціи для поиска ближайшей кофейни"
+        label.font = UIFont(name: "Verdana", size: 12)
+        label.textColor = #colorLiteral(red: 0.6120845675, green: 0.6320224404, blue: 0.6574426293, alpha: 1)
+        label.numberOfLines = 0
+        label.frame = CGRect(x: 30, y: 360, width: 260, height: 30)
+        return label
+    }()
+
+    private let labelPointView: UILabel = {
+        let label = UILabel()
+        label.text = "Адреса кофеен"
+        label.font = UIFont(name: "Verdana-Bold", size: 12)
+        label.frame = CGRect(x: 30, y: 340, width: 150, height: 15)
+        return label
+    }()
+
+    private let pointView: UIImageView = {
+        let image = UIImageView()
+        image.image = UIImage(named: "point")
+        image.frame = CGRect(x: 20, y: 330, width: 335, height: 70)
+        return image
+    }()
+
+    private let guestLabel: UILabel = {
+        let label = UILabel()
+        label.text = "Добро пожаловать, Гость"
+        label.font = UIFont(name: "Verdana-Bold", size: 16)
+        label.textColor = #colorLiteral(red: 0.8953791261, green: 0.86565727, blue: 0.8402896523, alpha: 1)
+        label.frame = CGRect(x: 20, y: 230, width: 185, height: 44)
+        label.numberOfLines = 0
+        return label
+    }()
+
+    private let cofeinovLabel: UIImageView = {
+        let cofeinov = UIImageView()
+        cofeinov.frame = CGRect(x: 100, y: 145, width: 165, height: 56)
+        cofeinov.image = UIImage(named: "cofeinov")
+        return cofeinov
+    }()
+
+    private let secondView: UIView = {
```

Comment:
```suggestion
    private let (привызываешь к объекту)View: UIView = {
```
таких секондвью может быть сотни